packages/strapi-plugin-users-permissions/admin/src/containers/AuthPage/index.js

```diff
@@ -143,67 +171,74 @@ export class AuthPage extends React.Component { // eslint-disable-line react/pre
   }
 
   renderInputs = () => {
-    const { match: { params: { authType } } } = this.props;
+    const { didCheckErrors, formErrors, match: { params: { authType } }, modifiedData, noErrorsDescription, onChangeInput, submitSuccess } = this.props;
     const inputs = get(form, ['form', authType]);
-
-    return map(inputs, (input, key) => (
-      <Input
-        autoFocus={key === 0}
-        customBootstrapClass={get(input, 'customBootstrapClass')}
-        didCheckErrors={this.props.didCheckErrors}
-        errors={get(this.props.formErrors, [findIndex(this.props.formErrors, ['name', input.name]), 'errors'])}
-        key={get(input, 'name')}
-        label={authType === 'forgot-password' && this.props.submitSuccess? { id: 'users-permissions.Auth.form.forgot-password.email.label.success' } : get(input, 'label')}
-        name={get(input, 'name')}
-        onChange={this.props.onChangeInput}
-        placeholder={get(input, 'placeholder')}
-        type={get(input, 'type')}
-        validations={{ required: true }}
-        value={get(this.props.modifiedData, get(input, 'name'), get(input, 'value'))}
-        noErrorsDescription={this.props.noErrorsDescription}
-      />
-    ));
+    
+    return map(inputs, (input, key) => {
+      const label = 
+        this.isAuthType('forgot-password') && submitSuccess 
```

Comment:
This condition is not really explicit, maybe we can put in a method `isForgotEmailSent` ?